index.html

```diff
@@ -5663,7 +5663,21 @@ <h3>Language and Base Direction</h3>
   "@id": "https://www.w3.org/2018/credentials/examples#title"
 }
         </pre>
+      </section>
+
+      <section class="informative">
```

Comment:
this says "informative", but the section contains a 2119 keyword `MAY`?

Reply:
Fixed.